src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/comparator/Car_Model_TokenizingJaccard_Comparator.java

```diff
@@ -0,0 +1,84 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.comparator;
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.Utils;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.Comparator;
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.Car;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.ComparatorLogger;
+import de.uni_mannheim.informatik.dws.winter.model.Correspondence;
+import de.uni_mannheim.informatik.dws.winter.model.Matchable;
+import de.uni_mannheim.informatik.dws.winter.model.defaultmodel.Attribute;
+import de.uni_mannheim.informatik.dws.winter.similarity.string.TokenizingJaccardSimilarity;
+
+
+public class Car_Model_TokenizingJaccard_Comparator implements Comparator<Car, Attribute> {
```

Comment:
Is this file still required? I think it does not make sense to check in samples like in the preprocess method.

Reply:
Its another similarity measure we can try - i will keep it in for now - it preprocesses the strings just like the other  comparator but uses a different measure - its to play with and for an easy comparison of the two measures